source/Timezones.mc

```diff
@@ -0,0 +1,114 @@
+using Toybox.WatchUi as Ui;
+using Toybox.System as Sys;
+using Toybox.Application as App;
+using Toybox.Graphics as Gfx;
+using Toybox.Time.Gregorian as Calendar;
+
+(:timezones)
+class Timezones extends Ui.Drawable {
+
+	var utcOffset = new Time.Duration(-Sys.getClockTime().timeZoneOffset);
+	
+	var tzLocations = [];
+	var UTC_Offsets = {};
+	var mTimeZones;
+	
+	var mhalfDCWidth; 
+	
+	var mTopY;
+	var mBottomY;
+
+    var faceWidth;
+    var faceHeight;
+
+	function initialize(params) {
+		Drawable.initialize(params);
+
+		mTopY = params[:topY];
+		mBottomY = params[:bottomY];
+		
+		setupTimezones();
+	}
+
+	function onSettingsChanged() {
+		setupTimezones();
+	}
+
+    function setupTimezones() {
+		tzLocations = [];
+		UTC_Offsets = {};
+        mTimeZones = App.getApp().getProperty("timezones").toNumber();
+        for (var i = 1; i <= mTimeZones; i++) {
+		    var name = App.getApp().getProperty("tz"+i+"_name");
+		    var offset = App.getApp().getProperty("tz"+i+"_offset").toNumber();
+		    var dst = App.getApp().getProperty("tz"+i+"_dst");
+		    if (name.equals("")) {
+			    continue;
+		    } else {
+			    tzLocations.add(name);
+		        System.println("Name:<"+name+">");	
+			    if (dst == true ) {
+				    offset = offset+1;
+			    }
+			    UTC_Offsets.put(name,offset);
+		    }
+	    }   
+    }
+	
+	function draw(dc) {
+		mhalfDCWidth = dc.getWidth() / 2;
+		drawTimezones(dc);
```

Comment:
Given that `draw()` is the only function to call `drawTimezones()`, it's probably worth inlining the former in the latter - this removes one member function, and saves memory (at the slight expense of readability).